Hostname:v2 docs (#28123)
keycloak/keycloak: docs/guides/server/hostname-deprecated.adoc

<#import "/templates/guide.adoc" as tmpl>
<#import "/templates/kc.adoc" as kc>
<#import "/templates/links.adoc" as links>

<@tmpl.guide
title="Configuring the hostname (v1)"
summary="Learn how to configure the frontend and backchannel endpoints exposed by {project_name}."
includedOptions="hostname hostname-* proxy"
deniedCategories="hostname_v2">

WARNING: The hostname configuration options used in this guide are deprecated and will be removed in a future release. Therefore, it is recommended to use the new hostname configuration options. For more details, see <@links.server id="hostname"/> guide.

== Server Endpoints

> {project_name} exposes different endpoints to talk with applications as well as to allow accessing the administration console. These endpoints

Reviewer: ```suggestion
{project_name} exposes different endpoints to talk with applications and allow access to the administration console. These endpoints
```
